### ISSUE TITLE
Notation: Hom

### ISSUE THREAD

**Issue by mauroporta** [mauroporta/Autour_de_DAG#3]
We should uniformize the hom notation. There are several of them:

(1) the classical hom of a category;
(2) the simplicial hom (or more generally an enriched hom);
(3) the internal hom of a closed category;
(4) the mapping space of a model category or of a \infty-category.

**Comment by pbelmans** [comment]
I don't know whether this is something we should do before a first submit or not. We should ask Gabriele. But at this point we shouldn't work too hard on this.

At some point we could make "an index of notation" using the package [`nomencl`](http://ctan.org/pkg/nomencl). If Mauro and I did this first (it's safe to say we are the most crazy and strict about notation :)) people could use this. On the other hand, large chunks (not quite the majority, but not far) seem to have been written by Mauro and me, so we might as well redo _all_ notation.

**Comment by mauroporta** [comment]
Yes, but since I still have to upload chapter 11, if we choose a standard notation I will write it in a conformal way, so that we don't have to modify it again.

Otherwise, nevermind, it won't be long in any case! I'll just define a shortcut for the hom set, so that we can modify from the preamble.

**Comment by pbelmans** [comment]
Changing your (internally) consistent notation to make it conform with the others will be less of an issue than making sure the less internally consistent exposés conform. So don't worry too much about it. At this moment it is more important that it is written than that it is consistent (I am sure it will look good, regardless of the notation used in other exposés).
